.ci-config/rippled.cfg

```diff
@@ -105,65 +105,66 @@ validators.txt
 # Note: The version of rippled you use this config with must have an implementation for the amendments you attempt to enable or it will crash.
 # If you need the version of rippled to be more up to date, you may need to make a comment on this repo: https://github.com/WietseWind/docker-rippled
 
-[amendments]
+[features]
```

Comment:
I thought the `features` stanza didn't work on startup which is why we needed the `amendments` stanza. When you run this locally are all these amendments enabled?

Reply:
Tests pass, so yes.